pom.xml

```diff
@@ -975,6 +975,7 @@ under the License.
     <version.sisu-maven-plugin>0.9.0.M4</version.sisu-maven-plugin>
     <version.plexus-utils>4.0.2</version.plexus-utils>
     <version.spotless-maven-plugin>3.0.0</version.spotless-maven-plugin>
+    <version.palantirJavaFormat>2.78.0</version.palantirJavaFormat>
```

Comment:
Is this version working with Java 17 (https://github.com/diffplug/spotless/issues/2451)?

Reply:
Even the newest 2.80.0 does.